### PR TITLE
Fix the syntax of one code block and change the wording

### PR DESCRIPTION
**Pull request by TheBlckbird**
I fixed the syntax of one code block I added in #845.
I also added one line break and fixed some grammar and typos.

### CODE REVIEW

**Comment by TheBlckbird** [comment]
Should be resolved

**Comment by TheBlckbird** [comment]
It might’ve been a mistake to use the same branch as in the other PR. I hope this works now

File: content/learn/book/getting-started/ecs/_index.md [hunk 1]
```diff
@@ -152,7 +152,8 @@ Marvelous!
 If we want to change the names of some people (perhaps they got married!), for example, we can do this using a mutable query:
 
 ```rs
-fn update_people(mut query: Query<&mut Name, With<Person>> {
+fn update_people(mut query: Query<&mut Name, With<Person>>) {
+
```
Comment (mockersf) [inline content/learn/book/getting-started/ecs/_index.md:156]:
```suggestion
```

Comment (TheBlckbird) [reply content/learn/book/getting-started/ecs/_index.md:156]: that wasn’t on purpose

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
Fixed in #850, which I got to first.

**Comment by TheBlckbird** [comment]
I'll create a new PR for the changed wording tomorrow.

**Pull request closed without merging**